core/controllers/library.py

```diff
@@ -70,9 +70,22 @@ class LibraryPage(base.BaseHandler):
     for search results.
     """
 
-    def get(self):
+    def get(self, **kwargs):
```

Comment:
Don't use kwargs -- this is a code smell, it's too 'open'. Instead of overloading this handler, how about making a new handler instead for the other pages -- say, LibraryGroupPage or something?